tidewave-core/src/server.rs

```diff
@@ -30,6 +30,29 @@ struct ShellParams {
     env: Option<HashMap<String, String>>,
 }
 
+#[derive(Deserialize)]
+struct ReadFileParams {
+    path: String,
+}
+
+#[derive(Deserialize)]
+struct WriteFileParams {
+    path: String,
+    content: String,
+    parents: Option<bool>,
```

Comment:
Do we have a case where we don't want to create the path?

Reply:
Yes, we can always create the parent. :)